SchedulerJobs/SchedulerJobs.UnitTests/Services/AnonymiseHearingsConferencesDataServiceTests.cs

```diff
@@ -71,7 +71,9 @@ public void Should_not_delete_vho_accounts()
         }
 
         [Test]
-        public void Should_ignore_not_found_errors_when_deleting_users()
+        [TestCase(404)]
+        [TestCase(403)]
```

Comment:
Would it be better to use enums here?

Reply:
I would have preferred if we could use the actual status code property here but you cant use that inside of TestCase. I am not sure if it would be any different if we use enum here instead of int

Reply:
Could you not use:

```suggestion
        [TestCase((int)HttpStatusCode.NotFound)]
        [TestCase((int)HttpStatusCode.Forbidden)]
```